scripts/insert_into_appointments.sql

```diff
@@ -1 +1,30 @@
 -- insert appointments creating connections between doctors and patients
+INSERT INTO appointments('date','time',comment,doctor_id,patient_id)
```

Comment:
`'date'` and `'time'` should be wrapped in double quotation marks here or nothing at all.  Also, the last two arguments are `doctor_id` and `patient_id`, but when you're inserting the data in the `SELECT` statements, you're inserting `p.id` and `d.id` in that order instead.